packages/header-bar/src/InnerHeader.js

```diff
@@ -59,7 +58,7 @@ class InnerHeader extends Component {
 
     componentWillReceiveProps(props) {
         if (this.props.lastUpdate && (this.props.lastUpdate.getTime() - props.lastUpdate.getTime()) !== 0) {
-            dhis2.menu.ui.bootstrapMenu();
+            dhis2.menu.ui.bootstrapMenu();  //eslint-disable-line
```

Comment:
Why does eslint need to be disabled here?

Reply:
because `dhis2` is apparently a global. Actually, I will see if this code is triggered

Reply:
It's better to hint to eslint that `dhis2` is a global:

```
/* global dhis2 */
```

Reply:
added /* global dhis2 */ and a check to be sure that it exists